systemtest/src/main/java/io/strimzi/systemtest/resources/KeycloakResource.java

```diff
@@ -0,0 +1,108 @@
+        String deploymentName = ResourceManager.kubeClient().getDeployment(KEYCLOAK_OPERATOR_DEPLOYMENT_NAME).getMetadata().getName();
+
+        // schedule deletion after teardown
+        ResourceManager.getPointerResources().push(() -> deleteKeycloakOperator(deploymentName, namespace));
+        // Wait for operator creation
+        waitForKeycloakOperator(deploymentName, namespace, 1);
+    }
+
+    public static void deployKeycloak(String namespace) {
+        ResourceManager.cmdKubeClient().namespace(namespace).apply(KeycloakResource.class.getResource("/keycloak/keycloak_instance.yaml").getFile());
+        waitForKeycloak(namespace);
+
+        // schedule deletion after teardown
+        ResourceManager.getPointerResources().push(() -> deleteKeycloak(namespace));
+    }
+
+    private static void deleteKeycloakOperator(String deploymentName, String namespace) {
+
+        try {
+            ResourceManager.cmdKubeClient().delete(FileUtils.downloadAndUnzip("https://raw.githubusercontent.com/keycloak/keycloak-operator/master/deploy/olm-catalog/keycloak-operator/9.0.2/keycloak-operator.v9.0.2.clusterserviceversion.yaml"));
+            ResourceManager.cmdKubeClient().delete(FileUtils.downloadAndUnzip("https://raw.githubusercontent.com/keycloak/keycloak-operator/master/deploy/olm-catalog/keycloak-operator/9.0.2/keycloakbackups.keycloak.org.crd.yaml"));
+            ResourceManager.cmdKubeClient().delete(FileUtils.downloadAndUnzip("https://raw.githubusercontent.com/keycloak/keycloak-operator/master/deploy/olm-catalog/keycloak-operator/9.0.2/keycloakclients.keycloak.org.crd.yaml"));
+            ResourceManager.cmdKubeClient().delete(FileUtils.downloadAndUnzip("https://raw.githubusercontent.com/keycloak/keycloak-operator/master/deploy/olm-catalog/keycloak-operator/9.0.2/keycloakrealms.keycloak.org.crd.yaml"));
+            ResourceManager.cmdKubeClient().delete(FileUtils.downloadAndUnzip("https://raw.githubusercontent.com/keycloak/keycloak-operator/master/deploy/olm-catalog/keycloak-operator/9.0.2/keycloaks.keycloak.org.crd.yaml"));
+            ResourceManager.cmdKubeClient().delete(FileUtils.downloadAndUnzip("https://raw.githubusercontent.com/keycloak/keycloak-operator/master/deploy/olm-catalog/keycloak-operator/9.0.2/keycloakusers.keycloak.org.crd.yaml"));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        ResourceManager.cmdKubeClient().namespace(namespace).delete(KeycloakResource.class.getResource("/keycloak/role.yaml").getFile());
+        ResourceManager.cmdKubeClient().namespace(namespace).delete(KeycloakResource.class.getResource("/keycloak/role_binding.yaml").getFile());
+        ResourceManager.cmdKubeClient().namespace(namespace).delete(KeycloakResource.class.getResource("/keycloak/service_account.yaml").getFile());
+
+        ResourceManager.cmdKubeClient().namespace(namespace).delete(KeycloakResource.class.getResource("/keycloak/operator.yaml").getFile());
```

Comment:
Hmmm, I think that I decided to this way because we are using a specific version of KO - `olm-catalog/keycloak-operator/9.0.2`. In that repository, they  have all these files for all versions and there could arise some inconsistency

Reply:
https://github.com/keycloak/keycloak-operator/blob/9.0.2/deploy/operator.yaml  hm?  You use 9.0.2 for all files, but master image in "local" operator data. I think we should use files from github if you don't need to change anything special here.